docs/source/conf.py

```diff
@@ -38,14 +39,18 @@
 
 RELEASE = os.environ.get("RELEASE", False)
 
-import pytorch_sphinx_theme
+sys.path.insert(0, os.path.abspath(".."))
+import pytorch_sphinx_theme2
 
 
+html_theme = "pytorch_sphinx_theme2"
+html_theme_path = [pytorch_sphinx_theme2.get_html_theme_path()]
+
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #
-needs_sphinx = "3.1.2"
+# needs_sphinx = "3.1.2"
```

Comment:
Why not delete this entire section? Because it uses sphinx-8 or something, aren't it?

Reply:
yes, will remove.